src/Components/Web.JS/@types/dotnet/dotnet.d.ts

```diff
@@ -170,8 +174,8 @@ interface AssetEntry extends ResourceRequest {
 }
 type AssetBehaviours = "resource" | "assembly" | "pdb" | "heap" | "icu" | "vfs" | "dotnetwasm" | "js-module-threads";
 type GlobalizationMode = "icu" | // load ICU globalization data from any runtime assets with behavior "icu".
-    "invariant" | //  operate in invariant globalization mode.
-    "auto";
+"invariant" | //  operate in invariant globalization mode.
+"auto";
```

Comment:
I looks like unintentional change - they are not aligned now.

Reply:
This file is auto-generated this way by the tool we use. This time I push exact unformatted copy.